src/scripts/scripts/zone/caverns_of_time/dark_portal/boss_temporus.cpp

```diff
@@ -141,12 +131,13 @@ struct boss_temporusAI : public ScriptedAI
         //Mortal Wound
         if (MortalWound_Timer < diff)
         {
-            canApplyWound = true;
-
-            if (m_creature->HasAura(SPELL_HASTE, 0))
-                MortalWound_Timer = urand(2000, 3000);
-            else
-                MortalWound_Timer = urand(6000, 9000);
+            AddSpellToCast(m_creature, SPELL_MORTAL_WOUND);
+                if (m_creature->HasAura(SPELL_HASTE, 0))
+                {
+                    MortalWound_Timer = urand(2000, 3000);
+                    else
```

Comment:
did your compiler accept this? xD
you should delete the {} brackets around this and put the else 4 spaces to the left